Sources/protoc-gen-swift/Context.swift

```diff
@@ -188,6 +197,20 @@ class Context {
   init(request: CodeGeneratorRequest) throws {
     self.request = request
     self.options = try GeneratorOptions(parameter: request.parameter)
+
+    if request.hasCompilerVersion {
+      let compiler_version = request.compilerVersion;
+      if (compiler_version.major == 3) &&
```

Comment:
Flipping the logic makes it hard to read (to me), so I used a local and negated the check of the local, better?